src/PrestaShopBundle/Translation/Provider/TranslationFinder.php

```diff
@@ -86,13 +86,13 @@ private function removeTrailingLocaleFromDomains(MessageCatalogueInterface $cata
 
     /**
      * @param string[] $paths
-     * @param string $pattern
+     * @param string|null $pattern
      *
      * @return Finder
      *
      * @throws FileNotFoundException
      */
-    private function getTranslationFilesFromPath(array $paths, string $pattern): Finder
+    private function getTranslationFilesFromPath(array $paths, ?string $pattern = null): Finder
```

Comment:
Maybe we should add some unit test to avoid a regression?

Reply:
Test added